src/ansys/mechanical/core/embedding/runtime.py

```diff
@@ -59,3 +73,16 @@ def initialize(version: int) -> None:
         # function codec is distributed with pymechanical on linux only
         # at version 242 or later
         __register_function_codec()
```

Comment:
remove the duplication. It is not a good user experience to throw this warning twice.

Reply:
It is not correct. __check_and_issue_pythonnet_warning does not throw an exception so I don't think the try/except will do anything

Reply:
still not correct. Here are the requirements:

1. using pythonnet is not an error
2. check for pythonnet/warn happens exactly once